src/slider/styled-components.js

```diff
@@ -38,18 +38,15 @@ Track.displayName = 'StyledTrack';
 export const InnerTrack = styled('div', props => {
   const {$theme, $value, $min, $max, $disabled} = props;
   const {colors, borders, sizing} = $theme;
+
   return {
     borderRadius: $theme.borders.useRoundedCorners ? borders.radius100 : '0px',
     background: getTrackBackground({
       values: $value,
       colors:
         $value.length === 1
-          ? [$disabled ? colors.mono600 : colors.primary, colors.mono400]
-          : [
-              colors.mono400,
-              $disabled ? colors.mono600 : colors.primary,
```

Comment:
Don't we want to keep color change in a disabled state?

Reply:
based on figma designs, disabled is indicated by drag handle color